packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# FIXME: remove this line when these tests don't hardcode bundle hashes.
+echo 1>&2 "FIXME: skipping zoe-full-upgrade tests"; return 0
+
```

Comment:
Can we move this at least after the `env_setup.sh` call below. I want to add to this upgrade test in an upcoming PR

Reply:
Sure, will do!